src/main/webapp/app/entities/statistics/statistics.component.html

```diff
@@ -11,24 +11,23 @@ <h2><span class="fas fa-chart-area"></span>&nbsp;<span>Statistics</span></h2>
     <div class="row">
         <div class="col-md-6">
             <h4><label for="reviewFullScreen" class="form-control-label">Report :</label></h4>
-            <select id="reviewFullScreen" class="form-control" [ngModel]="timeScale" (ngModelChange)="onSelectTimeScale($event)">
-                <optgroup label="Global overviews">
-                    <option value="default">Generated applications</option>
-                    <option value="clientFramework">Angular vs. React</option>
-                    <option value="buildTool">Maven vs. Gradle</option>
-                </optgroup>
-                <optgroup label="Time scale">
-                    <option value="daily">Last 24h</option>
-                    <option value="weekly">Last month</option>
-                    <option value="monthly">Last year</option>
-                </optgroup>
-            </select>
-        </div>
-        <div class="col-md-6 text-center">
-            <br/>
-            <button class="btn btn-primary" (click)="refreshData()">
-                  <fa-icon [icon]="'sync'"></fa-icon> <span>Refresh</span>
-            </button>
+            <span style="display: flex;">
```

Comment:
@ankitbeniwal : This will make the button appear misaligned on the right even for small screen sizes. So after this change we will have something like, 

<details>
<summary>After Change</summary>

![Screen Shot 2020-10-29 at 22 45 35](https://user-images.githubusercontent.com/12435965/97665385-87934b00-1a38-11eb-9aed-5d7b6a135cb2.png)
</details>

What we are trying to fix here is only the alignment of the button and select box in medium and larger screens, the alignment for small screens should stay as it is. Currently it appears at the bottom and that shouldn't change. 

<details>
<summary>Before Change</summary>

![Screen Shot 2020-10-29 at 22 51 38](https://user-images.githubusercontent.com/12435965/97665700-5b2bfe80-1a39-11eb-86ad-007afc02aeb8.png)
</details>

I suggest the following instead starting line 11.

```html
<div class="row">
        <div class="col-md-6">
            <select id="reviewFullScreen" class="form-control" [ngModel]="timeScale" (ngModelChange)="onSelectTimeScale($event)">
                <optgroup label="Global overviews">
                    <option value="default">Generated applications</option>
                    <option value="clientFramework">Angular vs. React</option>
                    <option value="buildTool">Maven vs. Gradle</option>
                </optgroup>
                <optgroup label="Time scale">
                    <option value="daily">Last 24h</option>
                    <option value="weekly">Last month</option>
                    <option value="monthly">Last year</option>
                </optgroup>
            </select>
        </div>
        <div class="col-md-6 text-center text-md-left p-2 p-md-0">
            <button class="btn btn-primary" (click)="refreshData()">
                <fa-icon [icon]="'sync'"></fa-icon> <span>Refresh</span>
            </button>
        </div>
</div>
```